backend/webapi/Features/AccessRequests/ImmsBC.cs

```diff
@@ -0,0 +1,108 @@
+namespace Pidp.Features.AccessRequests;
+
+using DomainResults.Common;
+using FluentValidation;
+using Microsoft.EntityFrameworkCore;
+using NodaTime;
+
+using Pidp.Data;
+using Pidp.Infrastructure.HttpClients.Keycloak;
+using Pidp.Infrastructure.HttpClients.Mail;
+using Pidp.Infrastructure.HttpClients.Plr;
+using Pidp.Infrastructure.Services;
+using Pidp.Models;
+using Pidp.Models.Lookups;
+
+public class ImmsBC
+{
+    public static IdentifierType[] AllowedIdentifierTypes => [IdentifierType.PhysiciansAndSurgeons, IdentifierType.Pharmacist, IdentifierType.Nurse];
```

Comment:
Unfortunately there is a bit of domain knowledge that the JIRA ticket glossed over here; don't worry that you didn't catch it.

The ticket asked for users to have a college licence from the college of Physicians and Surgeons / Nurses / Pharmacists. 
However, the college of Nurses and the College of Pharmacists actually have two Identifier Types each:
College of Nurses is both `Nurse` but also `Midwife`.
College of Pharmacists is both `Pharmacist` but also `Pharm Tech`.

It is rather difficult to find this knowledge in our code base without already knowing it; the one place we have it is in the `PlrClient` when we do a college licence search:
![image](https://github.com/user-attachments/assets/050dfc4c-b3d3-422b-91de-df328325b330)

I believe this is the first time we have had an enrolment that includes these two Colleges so there also isn't any code you could have based it on.